### PR TITLE
dual-pkg

### PR DESCRIPTION
**Pull request by tars0x9752**
#164
This is not an actual PR. Just to sharing the code to investigate the dual package hazard.

### CODE REVIEW

File: try-dual-pkg/lib-esm/index.mjs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Temporal } from '@js-temporal/polyfill';
```
Comment (12wrigja) [inline try-dual-pkg/lib-esm/index.mjs:1]: Could this just be 

`export * from '@js-temporal/polyfill';`

 ?

File: try-dual-pkg/lib-cjs/index.cjs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+function createZDT(year, month, day) {
+  return Temporal.ZonedDateTime.from({
+    timeZone: 'Asia/Tokyo',
+    year,
+    month,
+    day
+  });
+}
+
+module.exports = { createZDT };
```
Comment (12wrigja) [inline try-dual-pkg/lib-cjs/index.cjs:12]: Same here, but in some CJS-compatible complete re-export?

File: try-dual-pkg/consumer/index.mjs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+import { Temporal } from '@js-temporal/polyfill';
```
Comment (12wrigja) [inline try-dual-pkg/consumer/index.mjs:3]: Temporal as TemporalDirect ?

File: try-dual-pkg/consumer/index.mjs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import { createCurrentZDT, getDurationInHours } from 'lib-esm';
```
Comment (12wrigja) [inline try-dual-pkg/consumer/index.mjs:4]: Temporal as TemporalESMReExport

File: try-dual-pkg/consumer/index.mjs [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import { createZDT } from 'lib-cjs';
```
Comment (12wrigja) [inline try-dual-pkg/consumer/index.mjs:5]: Temporal as TemporalCJSReExport

**Comment by tars0x9752** [comment]
Closed because it does not have to be open.

**Pull request closed without merging**